docs/other/osg-token-renewer.md

```diff
@@ -0,0 +1,173 @@
+Installing and Using the OSG Token Renewal Service
+==================================================
+
+This document contains instructions to install and configure the
+OSG Token Renewal Service package, `osg-token-renewer`,
+for obtaining and automatically renewing tokens with the OIDC tools.
```

Comment:
```suggestion
for obtaining and automatically renewing tokens with [oidc-agent](https://github.com/indigo-dc/oidc-agent).
```

Reply:
Ok, i thought to say OIDC tools because it uses the other `oidc-*` commands, but i guess the package is called `oidc-agent`, so not far off.